website/docs/r/oracle_autonomous_database_cross_region_disaster_recovery.html.markdown

```diff
@@ -0,0 +1,122 @@
+
+## Example Usage
+
+```hcl
+resource "azurerm_oracle_autonomous_database_cross_region_disaster_recovery" "example" {
+  name                                = "example"
+  display_name                        = "example_display_name"
+  location                            = "westus"
+  database_type                       = "CrossRegionDisasterRecovery"
+  source                              = "CrossRegionDisasterRecovery"
+  source_id                           = "/subscriptions/xxxxx-xxxxxxxxxxxxx-xxxxxxxx/resourceGroups/resource_group_name/providers/Oracle.Database/autonomousDatabases/autonomous_database_name"
+  source_ocid                         = "ocid1.autonomousdatabase.xx.xx.xxxxxxxxxxxxxxx"
+  remote_disaster_recovery_type       = "Adg"
+  replicate_automatic_backups_enabled = true
+  subnet_id                           = "/subscriptions/xxxxx-xxxxxxxxxxxxx-xxxxxxxx/resourceGroups/resource_group_name/providers/Microsoft.Network/virtualNetworks/virtual_network_name/subnets/subnet_name"
+  virtual_network_id                  = "/subscriptions/xxxxx-xxxxxxxxxxxxx-xxxxxxxx/resourceGroups/resource_group_name/providers/Microsoft.Network/virtualNetworks/virtual_network_name"
+  resource_group_name                 = "example"
+  source_location                     = "eastus"
+  license_model                       = "example"
+  backup_retention_period_in_days     = 42
+  auto_scaling_enabled                = true
+  auto_scaling_for_storage_enabled    = true
+  mtls_connection_required            = false
+  compute_model                       = "example"
+  compute_count                       = 1.23456
+  db_workload                         = "example"
+  db_version                          = "example"
+  admin_password                      = "example"
+  character_set                       = "example"
+  national_character_set              = "example"
+}
+
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this Cross Region Disaster Recovery Autonomous Database. Changing this forces a new Cross Region Disaster Recovery Autonomous Database to be created.
+
+* `display_name` - (Required) The user-friendly name for the Autonomous Database. The name does not have to be unique.
+
+* `location` - (Required) The Azure Region where the Cross Region Disaster Recovery Autonomous Database should exist. Azure Region must be different from region where your original Autonomous Database resides. Changing this forces a new Cross Region Disaster Recovery Autonomous Database to be created.
+
+* `database_type` - (Required) The type of Autonomous Database. To create Cross Region Disaster Recovery Autonomous Database set this value to "CrossRegionDisasterRecovery".
```

Comment:
For the properties that are marked as "ForceNew". Please add "Changing this forces a new resource to be created." to the description. 

Please also check:
name
database_type
source
remote_disaster_recovery_type
admin_password
backup_retention_period_in_days
character_set
compute_model
db_version
db_workload
display_name
mtls_connection_required
license_model
national_character_set
subnet_id
virtual_network_id
customer_contacts